packages/api/src/typeDefs/collection.graphql

```diff
@@ -21,6 +21,9 @@ type StoreCollection {
   breadcrumbList: StoreBreadcrumbList!
   meta: StoreCollectionMeta!
   id: ID!
+  name: String!
   slug: String!
   type: StoreCollectionType!
+  level: Int
+  parentId: String
```

Comment:
Is it a number or a string? Not clear from the way you typed on TS and here on the schema

Reply:
It should be the same of the `id` field, I think. In TS, `id` is a number, but in the schema it returns a string, so I set `parentId` as a string there too